new-log-viewer/src/services/LogFileManager.ts

```diff
@@ -152,6 +153,30 @@ class LogFileManager {
         this.#decoder.setDecoderOptions(options);
     }
 
+    loadChunk (eventIdx: number): {
+        logs: string,
+    } {
```

Comment:
Since we only return one string from this method, shall we directly return a string instead of an object containing the string?
```suggestion
    loadChunk (eventIdx: number): string {
```

Reply:
I see how the return value of this method is directly used by the main service worker. I think it's fine to leave the return type unchanged if that's intended.